### DISCUSSION TITLE
如何在编译后复制文件夹

### DISCUSSION THREAD

**Discussion by HikariOri** [xmake-io/xmake#3903] [Q&A] [answered]
例如有一个 `shaders` 文件夹，编译生成的可执行文件在 `build\windows\x64\release` 目录下，我希望在编译完成后将 `shaders` 文件夹和里面的文件全部复制到 `build\windows\x64\release` 下，请问该如何做

**Comment by Ligo04** [comment, marked as answer]
https://xmake.io/#/zh-cn/manual/builtin_modules?id=oscp 
使用os.cp将文件复制过去
